tests/longevity/longevity_test.go

```diff
@@ -447,6 +450,17 @@ func populateIntervals() {
 	triggerInterval[RebootNode][2] = 24 * baseInterval
 	triggerInterval[RebootNode][1] = 27 * baseInterval
 
+	triggerInterval[CrashNode][10] = 1 * baseInterval
+	triggerInterval[CrashNode][9] = 3 * baseInterval
+	triggerInterval[CrashNode][8] = 6 * baseInterval
+	triggerInterval[CrashNode][7] = 9 * baseInterval
+	triggerInterval[CrashNode][6] = 12 * baseInterval
+	triggerInterval[CrashNode][5] = 15 * baseInterval
+	triggerInterval[CrashNode][4] = 18 * baseInterval
+	triggerInterval[CrashNode][3] = 21 * baseInterval
+	triggerInterval[CrashNode][2] = 24 * baseInterval
```

Comment:
can you please add triggerinterval 0 for crashnode. it is used to disable the test if needed